packages/vue-cli-plugin-vuetify/generator/tools/fonts.js

```diff
@@ -7,6 +7,11 @@ const fonts = {
     import: '@mdi/font/css/materialdesignicons.css',
     link: '<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@mdi/font@latest/css/materialdesignicons.min.css">',
   },
+  mdiSvg: {
+    package: {
+      '@mdi/js': '^4.7.95',
```

Comment:
Could be `'*'`, they are not following semantic versioning

Reply:
Good call. I've updated the PR to reflect this change.